### PR TITLE
Project 3: Business site "Surfboard Rental"

### PR DESCRIPTION
**Pull request by Thessan**

### CODE REVIEW

**Review by bealun — APPROVED** [review]
Amazing work this week! 👏 You've done a really cool site, super responsive and with a nice code structure. Keep it up, it's very nice to see! I do wanna encourage you to be more consistent with you indentation and also to push your code more frequently to GitHub, with informative commit messages. This will only benefit you and your developer journey!
You should be SO proud of yourself and your work 🎉

File: code/style.css [hunk 1]
```diff
@@ -1,7 +1,159 @@
...
+  background-color: rgb(214, 214, 214);
+  font-family: 'Times New Roman', Cambria, Cochin, Georgia, Times;
+  font-size: 15px;
+  margin: 0;
+ }
 
-.active + .answer {
-  /* some styling here */
+ /* I'm aware that there is a gap on the right between the image and the browser window,
+ haven't been able to figure out how to fix this */
```
[the remark this answers is not in the document]
Comment (Thessan) [reply code/style.css:10]: Thank you, fixed it! :)

File: code/style.css [hunk 2]
```diff
@@ -1,7 +1,159 @@
...
+.hero-text {
+display: flex;
+flex-direction: column;
+position: absolute;
+color: rgb(35, 141, 190);
+border: double 10px rgba(255, 255, 255, 0.945);
+background-color: rgba(255, 255, 255, 0.404);
+padding: 15px;
+margin: 10px;
+}
```
[the remark this answers is not in the document]
Comment (Thessan) [reply code/style.css:35]: Im not sure I understand exactly what you mean, can you please clarify?:)

[the remark this answers is not in the document]
Comment (bealun) [reply code/style.css:35]: This just needs to be indented one level, like in the rest of your CSS - that looks good! So a reminder to be consistent with your structure all over, which will benefit you.

**Pull request closed without merging**